src/ODEmodel.jl

```diff
@@ -62,16 +62,17 @@ function predict(p, g_0::Real, t, nG1::Integer, nG2::Integer, nD1, nD2, retVec =
         D2 = zeros(nD2)
     end
 
-    v = [ones(nG1) * p[5] * g_0 / nG1; ones(nG2) * (1.0 - p[5]) * g_0 / nG2; D1; D2]
     A = ODEjac(p, nG1, nG2, nD1, nD2)
 
     if t isa Real
-        v = ExponentialUtilities.expv(t, A, v)
-        G1 = sum(v[1:nG1]) + sum(v[(nG1 + nG2 + 1):(nG1 + nG2 + nD1)])
-        G2 = sum(v[(nG1 + 1):(nG1 + nG2)]) + sum(v[(nG1 + nG2 + nD1 + 1):(nG1 + nG2 + nD1 + nD2)])
+        G1 = sum(vec[1:nG1]) + sum(vec[(nG1 + nG2 + 1):(nG1 + nG2 + nD1)])
+        G2 = sum(vec[(nG1 + 1):(nG1 + nG2)]) + sum(vec[(nG1 + nG2 + nD1 + 1):(nG1 + nG2 + nD1 + nD2)])
```

Comment:
Why are you taking the sum of G1 and G2 before `expv`? This is just the starting vector you passed in.

Reply:
The vector it is taking comes from the output of the `else` part, which is already A*v. I thought if I put the `expv` above, I would double exponentiate it.

Reply:
What is the vector being passed in here? The way to think about this is not where it has been exponentiated, but what it represents.

Reply:
This is how I thought about it:
First, we call predict for the first 48 hours for lapatinib with a time vector and return the state vector at the last time point.
Then we pass in the state vector so that the `if statement` is true, so it will produce the first data point after 48 hours. We loop through this and successively pass the produced state vector to build the next data point and state vector.

Reply:
Ahh... you don't ever need to use the `ExponentialUtilities.expv` part of this function for the temporal combination. The **only** reason I added this is for when you only need to calculate one time-point. The **only** change you need for temporal combinations is to be able to pass in the starting state vector. The trick is, if you're switching drugs, t=48hrs is effectively t=0 as far as predict is concerned.

Reply:
Oh, I see. Thank you. I'll make the changes right away.